src/pages/home/ReportScreen.tsx

```diff
@@ -161,6 +163,29 @@ function ReportScreen({
     const isLoadingReportOnyx = isLoadingOnyxValue(reportResult);
     const permissions = useDeepCompareRef(reportOnyx?.permissions);
 
+    // check if there's a reportID in the route. If not, set it to the last accessed reportID
+    const lastAccessedReportID = useLastAccessedReportID(!!route.params.openOnAdminRoom);
+    useEffect(() => {
+        // Don't update if there is a reportID in the params already
+        if (route.params.reportID) {
+            const reportActionID = route?.params?.reportActionID;
+            const regexValidReportActionID = new RegExp(/^\d*$/);
```

Comment:
NAB we could use ValidationUtils.isNumeric
```suggestion
            const isValidReportActionID = ValidationUtils.isNumeric(reportActionID);
```

Reply:
This was existing code, but in for a penny in for a pound I suppose 😉